sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -15,93 +15,131 @@
 		final ArrayList<NameValuePair> nameValues = new ArrayList<NameValuePair>();
-	    nameValues.add(new BasicNameValuePair("Authorization", "Bearer"+ " " +getCanvasAdminToken()));
-	    nameValues.add(new BasicNameValuePair("content-type", "application/json"));
-	    for (final NameValuePair h : nameValues)
-	    {
-	        clientRequest.addHeader(h.getName(), h.getValue());
-	    }
+		nameValues.add(new BasicNameValuePair("Authorization", "Bearer"+ " " +canvasToken));
+		nameValues.add(new BasicNameValuePair("content-type", "application/json"));
+		for (final NameValuePair h : nameValues)
+		{
+		    clientRequest.addHeader(h.getName(), h.getValue());
+		}
 		BufferedReader rd = null;
 		try {
 			 rd = new BufferedReader(new InputStreamReader(client.execute(clientRequest).getEntity().getContent()));
-		} catch (ClientProtocolException e) {
-			e.printStackTrace();
 		} catch (IOException e) {
-			e.printStackTrace();
+			M_log.error("Canvas API call did not happen",e);
 		}
 		String line = "";
 		StringBuilder sb = new StringBuilder();
 		while ((line = rd.readLine()) != null) {
 			sb.append(line);
-			System.out.println(sb.toString());
 		}
-		request.setCharacterEncoding("UTF-8");
 		response.setContentType("application/json");
 		PrintWriter out = response.getWriter();
 		out.print(sb.toString());
 		out.flush();
 		
 		}
-		 //TODO Delete
-		String contextPath = request.getContextPath();
-		String requestURI = request.getRequestURI();
-		Enumeration headerNames = request.getHeaderNames();
-		Map<String,String> parameterMap = request.getParameterMap();
-		Enumeration<String> parameterNames = request.getParameterNames();
-		String pathTranslated = request.getPathTranslated();
-		String remoteAddr = request.getRemoteAddr();
-		String localAddr = request.getLocalAddr();
-		StringBuffer requestURL = request.getRequestURL();
-		String serverName = request.getServerName();
-		System.out.println("ServerName: "+serverName);
-		System.out.println("ServletPath: "+servletPath);
-		System.out.println("RequestURL: "+requestURL.toString());
-		System.out.println("Remote Address: "+remoteAddr);
-		System.out.println("PathTranslated: "+pathTranslated);
-		System.out.println("LocalAddr: "+localAddr);
-		System.out.println("PathInfo: "+pathInfo);
-		System.out.println("ParameterNames: "+parameterNames);
-		System.out.println("ParameterMap: "+parameterMap);
-		System.out.println("HeaderNames: "+headerNames);
-		System.out.println("RequestURI: "+requestURI);
-		System.out.println("QueryString: "+queryString);
-		System.out.println("ContextPath: "+contextPath);
 	}
+
 	
-	private String getCanvasAdminToken() {
-		return "REPLACE WITH ACTUAL TOKEN";
+	protected void getCanvasCredentials() {
+		M_log.debug("getCanvasCredentials(): called");
+		String propertiesFilePath = System.getProperty(SYSTEM_PROPERTY_FILE_PATH);
+		if (!isEmpty(propertiesFilePath)) {
+		canvasProperties=PropertiesUtilities.getPropertiesObjectFromURL(propertiesFilePath);
```

Comment:
I need to use better name. i will clean up that. 

I have a external properties file some where on linux box in a directory, and the properties file has list of properties like 
canvas.token=asbjdbs
canvas.url =http://test.canvas.com
use.test.url =true 
ldap.server.name = ldap.server.umich.edu

why do need to place the just canvas related properties in another separate file?